public/app/app.js

```
@@ -2050,21 +2225,41 @@ class VoiceIsolatePro {
     if (this.dom?.tpABLabel) this.dom.tpABLabel.textContent = 'Processed';
   }
 
+  _resetSliders({ unlockedOnly = true } = {}) {
+    qsa('[id^="sl_"]').forEach((el) => {
```

ESLint

[error] 'qsa' is not defined.

(no-undef)